src/dlmalloc.rs

```diff
@@ -225,7 +230,7 @@ impl Dlmalloc {
     }
 
     pub unsafe fn calloc_must_clear(&self, ptr: *mut u8) -> bool {
-        !sys::allocates_zeros() || !Chunk::mmapped(Chunk::from_mem(ptr))
+        !Sys::allocates_zeros(&self.system_allocator) || !Chunk::mmapped(Chunk::from_mem(ptr))
```

Comment:
I think this (and other callsites to other methods) can be switched now to use `self.system_allocator.allocates_zeros()` (using `.` for method calls rather than the UFCS form)